projects/netgrif-components-core/src/lib/navigation/navigation-double-drawer/abstract-navigation-double-drawer.ts

```diff
@@ -179,7 +179,7 @@ export abstract class AbstractNavigationDoubleDrawerComponent implements OnInit,
             return;
         }
         if (node.parentId && !node.parent) {
-            if (node.parentId === this._uriService.root.id) {
+            if (node.parentId === this._uriService.root.path) {
```

Comment:
Since the id attribute of UriNodeResource is gone, what values are represented by `node.parentId`? Is this condition be ever true? Also consider using `uriService.isRoot(node)`

Reply:
The attribute `parentId` comes from backend, setter logic is implemented there for this attribute.  `uriService.isRoot(node)` added.